### DISCUSSION TITLE
transfer holidays over to next year

### DISCUSSION THREAD

**Discussion by pkoevesdi** [kimai/kimai#5240] [Plugins]
One of our employees asked, what happens with not taken holidays at the end of the year? Are they removed, and have to be taken over as a manual booking, or does something like that happen automatically?

**Comment by pkoevesdi** [comment]
Ok, seems nothing happened automatically. Neither for the Plus-hours. I'd like to point both as a feature request.

**Reply by blattms** [reply]
Same for the public holiday calendar. You need to recreate these for the new year, too
